probe.go

```diff
@@ -670,6 +670,121 @@ func probeLicenseStatus(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		linkStatus = prometheus.NewDesc(
+			"fortigate_link_status",
+			"signals if the status is up (1) or down (0)",
+			[]string{"vdom", "link_group", "link"}, nil,
```

Comment:
I would like to expose that these metrics come from the "link monitor" subsystem somehow. Adding it to the metric name seems wrong, it would be too long and annoying. I was thinking of that maybe we can rename the `link_group` to be `monitor` - that way it would be something like:
`fortigate_link_latency_seconds{monitor="google-dns",...} 0.001`

I think that looks pretty neat, what do you think?

Reply:
I have no direct access to a fortigate so some of the terminology is a mystery for me.   If it is called like that in the UI this would be a better fit.